geopandas/io/tests/test_file.py

```diff
@@ -275,6 +273,28 @@ def test_append_file(tmpdir, df_nybb, df_null, driver, ext):
     assert_geodataframe_equal(df, expected, check_less_precise=True)
 
 
+@pytest.mark.xfail(strict=False)
+@pytest.mark.parametrize("driver,ext", driver_ext_pairs)
+def test_gpkg_empty_crs(tmpdir, driver, ext):
+    """Test handling of undefined CRS with GPKG driver (GH #1975)."""
+    tempfilename = os.path.join(str(tmpdir), "boros." + ext)
+    df = GeoDataFrame(
+        {
+            "a": [1, 2, 3],
+            "geometry": [Point(0, 0), Point(1, 1), Point(2, 2)],
+        },
+    )
+
+    df.to_file(tempfilename, driver=driver)
+    result = read_file(tempfilename)
+
+    if driver == "GeoJSON":
+        # geojson by default assumes epsg:4326
+        result.crs = None
```

Comment:
```suggestion
        df.crs = "EPSG:4326"
```

(then it actually tests that it is set like that, although that might already be covered elsewhere)